pom.xml

```diff
@@ -413,6 +417,61 @@
       </build>
     </profile>
 
+    <profile>
+      <id>java-next</id>
```

Comment:
Cool name, but is there a reason for `java-next` instead of `java-9`? I actually guess that with the timelines we might be testing with Java 8, 9, 10 at the same time.

Reply:
Well the thing I was thinking is that we probably should define our target if we will go for the whole spectrum (9.10,11) or if we asume like the milestone will be the LTS version (11). I will rename it to 9 and we will see how we evolve it depending pn the future releases.